package.json

```diff
@@ -50,6 +53,7 @@
     "oref0-meal": "./bin/oref0-meal.js",
     "oref0-mint-max-iob": "./bin/oref0-mint-max-iob.sh",
     "oref0-normalize-temps": "./bin/oref0-normalize-temps.js",
+    "oref0-ns-autotune": "./bin/oref0-ns-autotune.sh",
```

Comment:
i would rename `bin/oref0-autotune.js` to  `bin/oref0-autotune-core.js` and use `bin/oref0-autotune` for the cli program. That NS is used as a source confuses me, because you want the name to be clear about what it is auto tuning. And that is oref0.